api/src/wfl/source.clj

```diff
@@ -3,9 +3,9 @@
             [clojure.instant       :as instant]
             [clojure.spec.alpha    :as s]
             [clojure.set           :as set]
-            [clojure.tools.logging :as log]
```

Comment:
Old log module, must have been missed when switching out.  I was initially not seeing the logs I expected out of this namespace.